includes/data-port/class-sensei-data-port-utilities.php

```diff
@@ -105,6 +105,9 @@ public static function get_attachment_from_source( $source, $parent_id = 0 ) {
 	 * @return int|WP_Error  The attachment id or an error.
 	 */
 	public static function create_attachment_from_url( $external_url, $parent_id = 0 ) {
+		require_once ABSPATH . 'wp-admin/includes/image.php';
+		require_once ABSPATH . 'wp-admin/includes/file.php';
+		require_once ABSPATH . 'wp-admin/includes/media.php';
 
```

Comment:
Is this used for this PR?

Reply:
Sorry, forgot to put it in the changelog. This fixes an issue when WP Cron is running the job instead of Action Scheduler.